src/main/java/org/commcare/cases/instance/IndexedFixtureInstanceTreeElement.java

```diff
@@ -0,0 +1,70 @@
+package org.commcare.cases.instance;
+
+import org.commcare.cases.model.StorageIndexedTreeElementModel;
+import org.commcare.core.interfaces.UserSandbox;
+import org.commcare.modern.util.Pair;
+import org.javarosa.core.model.instance.AbstractTreeElement;
+import org.javarosa.core.model.instance.InstanceBase;
+import org.javarosa.core.services.storage.IStorageUtilityIndexed;
+import org.javarosa.model.xform.XPathReference;
+import org.javarosa.xpath.expr.XPathPathExpr;
+
+import java.util.Hashtable;
+
+/**
+ * The root element for the a indexed fixture data instance: instance('some-indexed-fixture').
```

Comment:
for clarify in the documentation, this element represents 

instance('some-indexed-fixture')/fixtureroot

doesn't it?